### DISCUSSION TITLE
Horrible usability when editing an unpublished post or page

### DISCUSSION THREAD

**Discussion by dev12-byte** [elementor/elementor#28171] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

When creating a new post you can select between private, publish, pending and a few more depending on the WP version. Whenever you select anything other than "Published", editing the page in elementor is very confusing. 

Example:
You created a post with the status "pending". When you now open the editor the biggest and most primary button on the top right says "publish" or "update". This is normally not an issue as you are usually editing a published page and therefore you can interpret the button as "save". 
However when editing a non-published post this button will IMMEDIATLY publish the post although you set the post status to "pending".  Only when you select the dropdown you can only save the post (Which is also not ideal as it says "save as draft" and you are not always selecting type "draft")

And it is extreme confusing as you often click this button out of habit. 

### Describe the solution you'd like.

Whenever the post or page status is not set as "published", the primary button should say "Save" or "Update". In the dropdown you can then publish the post by clicking "Publish now". Alternatively, adding a second button next to the primary one, where the highlighted one is for saving and the secondary one is for publishing would also work

### Describe alternatives you've considered.

If all that is to complex please just add a confirmation message to the button. This message should always appear if clicking the button would change the status of the post from anything to published. 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by arsanet** [comment]
EXACTLY! We can't stay in draft state when updating post/page with Elementor.
